src/ovp_pipeline/commands/source_coverage.py

```
@@ -0,0 +1,321 @@
+    """Render top-N unrecognized hosts as a YAML stub for paste-into-overrides."""
+    lines = [
+        "# Triage stub — review each entry's `authority` and `bucket`,",
+        "# delete the ones you don't want, then paste into",
+        "# 60-Logs/domain_overrides.yaml under `domains:`.",
+        "domains:",
+    ]
+    for s in stats[:top_n]:
+        sample = s.sample_urls[0] if s.sample_urls else ""
+        lines.append(f"  {s.host}:")
+        lines.append(f"    authority: 0.55  # TODO: review (sample: {sample})")
+        lines.append(f"    bucket: mixed    # TODO: canonical | mixed | low")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)